csharp-selenium-webdriver-sample/SampleTest.cs

```diff
@@ -6,57 +6,74 @@
 using System;
 using System.IO;
 using Selenium.Axe;
-using OpenQA.Selenium.Chrome;
 
 namespace CSharpSeleniumWebdriverSample.Test
 {
     [TestClass]
-    [DeploymentItem("SamplePage.html")]
-    [DeploymentItem("chromedriver.exe")]
-    [DeploymentItem("geckodriver.exe")]
+   
     public class SampleTest
     {
+        // Instance to control the browser.
+        // The OpenQA.Selenium.IWebDriver interface is the main interface to use for testing, which represents an idealized web browser.
         private IWebDriver _webDriver;
 
         [TestCleanup]
         public virtual void TearDown()
         {
+            // You should quit the driver on TearDown which will close every associated window.
             _webDriver?.Quit();
         }
 
 
         [TestMethod]
+        // First DataRow Attribute will create a new instance of the test method and pass Chrome as a paramter, so it'll test on ChromDriver
```

Comment:
spelling: "paramter" -> "parameter" (there are a few other instances of this to ctrl+F for)